public/app/ui/components/search/results/search-results.html

```diff
@@ -36,8 +36,8 @@
                     <tbody>
                     <!-- ko foreach: { data: results, as: 'result' } -->
                     <tr class="result">
-                        <td data-bind="text: $index() + $parent.pager.start() + 1"></td>
-                        <!-- ko foreach: { data: $parent.fields, as: 'field' } -->
+                        <td data-bind="text: $index() + $parents[1].pager.start() + 1"></td>
```

Comment:
OK it took me a bit longer to understand this one but it makes sense now, and I don't this is the right fix (although of course it does work).  

The `pager` property is on the  `SearchPage` view model.  Previously, this code was in `table-results.html` which was included as a `component` binding that passed the `SearchPage` through (as `$data`), so in other words, the fact that the code was in `table-results.html` and not `search.html` was effectively transparent, so `$parent` was correct (it's iterating thorugh each element of `results` so you need to go one level up to get to the view model).  

Now, this is in `search-results.html` which is bound to a separate view model, `SearchResultsComponent`.  So by using `$parents[1]` you're going above the `SearchResultsComponent` to the containing `SearchPage`.

This is incorrect because it's invisibly coupling the two view models together.  Better to pass the `start` observable to the `SearchResultsComponent`.

I can update this PR or @kehh do you want to make this change?